src/NotFoundPage.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function NotFoundPage() {
+  return (
+    <p>404 Not Found</p>
+  )
+}
```

Comment:
```suggestion
}

// Some other code
```

코드 마지막 줄에는 빈 줄이 하나가 추가되어야 합니다. 그렇지 않으면 위와 같이 GitHub에서 표시가 되고 또한 eslint에서 통과하지 못합니다. 

빈 줄을 추가해야하는 이유는 GitHub의 컨벤션 때문에 그렇습니다. GitHub은 변경사항을 표시할 때 변경전 부분과 변경 후 를 모두 보여줍니다. 


```js
console.log('Hello');
```

만약 위와 같이 빈 줄이 없는 코드에서 아래에 코드를 추가하게 되면 Git은 다음과 같이 표시합니다.

```js
- console.log('Hello');
+ console.log('Hello');
+ console.log('world!');
```

저는 단순히 아래 한 줄만 추가했는데 왜 세줄이 변경되었다고 나오는 걸까요? 그 이유는 줄바꿈 문자 \n가 들어갔기 때문입니다. 


```js
- console.log('Hello');
+ console.log('Hello');\n
+ console.log('world');
```

만약 미리 빈 줄을 하나 추가한 상태에서 아래에 새로운 코드를 작성할 경우에는 다음과 같이 보여집니다.

```js
+ console.log('world');
```

이미 \n이 추가되어 있기 때문이죠. 따라서 코드 마지막 줄에 빈 줄 하나를 항상 추가해주는게 좋습니다.

Reply:
@hannut91 린트에서 빈 한줄을 계속 체크하는게 이런 이유였군요. 감사합니다. 반영하였습니다.